.pylintrc

```diff
@@ -278,7 +278,7 @@ exclude-too-few-public-methods=
 ignored-parents=
 
 # Maximum number of arguments for function / method.
-max-args=5
+max-args=7
```

Comment:
BASED

Reply:
This doesn't help us, and increasing doesn't really hurt us